packages/docs-site/docs/ref/domain/usage.md

```diff
@@ -1,4 +1,4 @@
-# Syntax
+# Usage
```

Comment:
For consistency, since your updated header in the Style docs says "Style Usage":
```suggestion
# Domain Usage
```